src/content/en/updates/2017/09/chrome-62-media-updates.md

```diff
@@ -0,0 +1,173 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the media (audio/video) updates in Chrome 62.
+
+{# wf_updated_on: 2017-09-08 #}
+{# wf_published_on: 2017-09-08 #}
+{# wf_tags: news,chrome62,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Blink>Media #}
+
+# Media (Audio/Video) Updates in Chrome 62 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+- [Offline playback with DRM licenses](#offline-playback) is now supported on
+  Android.
+- Chrome now [disables video tracks when a MSE video is played in the
+  background](#background-video-track-optimizations) to optimize performance.
+- Web developers can [customize seekable range](#seekable)
+  on live MSE streams.
+- Chrome now supports [FLAC in MP4 with MSE](#flac-in-mp4-with-mse).
+- Video will [go fullscreen when device is rotated](#auto-fullscreen-rotate).
+
+- TODO: Media preload defaults to metadata
+
+## Persistent licenses for Android
+
+Persistent license in [Encrypted Media Extensions (EME)] means the license can
+be persisted on the device, so that application can load the license into
+memory without sending another license request to the server. This is how
+offline playback is supported in EME.
+
+Until now, Chrome OS was the only platform to support persistent licenses. It
+is not true anymore. Playing protected content through EME while device is
+offline is now possible on Android as well.
+
+    const config = [{
+      sessionTypes: ['persistent-license'],
+      videoCapabilities: [{
+        contentType: 'video/webm; codecs="vp9"',
+        robustness: 'HW_SECURE_ALL' // Widevine specific
+      }]
+    }];
+
+    // Chrome will prompt user if website is allowed to uniquely identify
+    // user's device to play protected content.
+    navigator.requestMediaKeySystemAccess('com.widevine.alpha', config)
+    .then(access => {
+      // User will be able to watch encrypted content while being offline when
+      // license is stored locally on device and loaded later.
+    })
+    .catch(error => {
+      // Persistent licenses are not supported on this platform yet.
+    });
+
+You can try it yourself by checking out the [Sample Media PWA] and following
+these steps:
+
+1. Go to [https://biograf-155113.appspot.com/ttt/episode-2/]
+2. Click "Make available offline" and wait for video to be downloaded
```

Comment:
Change 'wait for video' to 'wait for the video'.

Reply:
Done.